openedx/core/djangoapps/api_admin/models.py

```diff
@@ -88,3 +98,75 @@ class ApiAccessConfig(ConfigurationModel):
 
     def __unicode__(self):
         return u'ApiAccessConfig [enabled={}]'.format(self.enabled)
+
+
+@receiver(post_save, sender=ApiAccessRequest, dispatch_uid="api_access_request_post_save_email")
+def send_request_email(sender, instance, created, **kwargs):   # pylint: disable=unused-argument
+    """ Send request email after new record created. """
+    if created:
+        _send_new_pending_email(instance)
+
+
+@receiver(pre_save, sender=ApiAccessRequest, dispatch_uid="api_access_request_pre_save_email")
+def send_decision_email(sender, instance, **kwargs):  # pylint: disable=unused-argument
+    """ Send decision email after status changed. """
+    if instance.id and not instance.contacted:
+        old_instance = ApiAccessRequest.objects.get(pk=instance.id)
+        if instance.status != old_instance.status:
+            _send_decision_email(instance)
+
+
+def _send_new_pending_email(instance):
+    """ Send an email to settings.API_ACCESS_MANAGER_EMAIL with the contents of this API access request. """
+    context = {
+        'approval_url': urljoin(
+            instance.base_url,
+            reverse('admin:api_admin_apiaccessrequest_change', args=(instance.id,))
```

Comment:
@clintonb this is the default django admin namespace.  This is the edit link for marketing.  Some quick searching makes me think that we either can't or don't want to mess with it